src/lib/compatibility-check.js

```diff
@@ -1,6 +1,9 @@
+export const isNode = typeof global !== 'undefined' && ({}).toString.call(global) === '[object global]';
 const arrayIndexOfSupport = typeof Array.prototype.indexOf !== 'undefined';
 const postMessageSupport = typeof window.postMessage !== 'undefined';
```

Comment:
I have a feeling the `typeof window.postMessage` check will throw if there is now window, causing https://github.com/vimeo/player.js/issues/32#issuecomment-237671072. Probably needs a `typeof window !== undefined` check in there as well?